gui/api/resources.py

```diff
@@ -738,109 +738,109 @@ def obj_get(self, bundle, **kwargs):
     def prepend_urls(self):
         return [
             url(
-                r"^(?P<resource_name>%s)/(?P<pk>\w[\w/-]*)/datasets%s$" % (
```

Comment:
I am not sure why we allowed forward slashes in the regex before. Would you know if we had a reason for it before and if it's okay to keep on allowing it ?

Reply:
It has always been like this, I don't know for which reason, but for sake of not breaking anything let's keep it.

Reply:
Thank you for clarifying :)